lib/SimpleHttpClient/Communicator.cpp

```diff
@@ -393,6 +393,7 @@ void Communicator::createRequestInProgress(NewRequest const& newRequest) {
 
 void Communicator::handleResult(CURL* handle, CURLcode rc) {
   // remove request in progress
+  double connectTime(0.0);
```

Comment:
This is never set to anything non-zero, and only once later checked whether it is zero. This seems superfluous.

Reply:
see comment below.